### PR TITLE
added single video page

### PR DESCRIPTION
**Pull request by nikhiljugale007**
files to check:
SingleVideo.jsx
SingleVideo.css
HzVideoCard.jsx
HzVideoCard.css
VideoPlayer.jsx
VideoPlayer.css

### CODE REVIEW

**Review by thenorthkun — APPROVED** [review]
Well written code, good to merge 👍

**Review by rajaranjanmr — APPROVED** [review]
everything seems fine , good to merge.

File: src/pages/single-video/SingleVideo.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.video-description-container {
+	display: flex;
+	justify-content: space-between;
+	padding: 1rem;
+}
+.btn-danger {
+	height: fit-content;
+}
+
+@media only screen and (max-width: 1024px) {
```
Comment (Sahil25061999) [inline src/pages/single-video/SingleVideo.css:28]: you can use em here for media query

**Review by Sahil25061999 — APPROVED** [review]
everything looks good, just suggested a minor change. 
UI suggestion: I think the cards in the homepage can be a bit more spaced, also the card top corners can also be rounded. 
The searchbar width can be a bit longer, also it will be nice if it also has rounded corner will give a consistent look, also the color for the search button can be a little dark.

File: src/pages/single-video/SingleVideo.css (hunk 1 above)
Comment (nikhiljugale007) [reply src/pages/single-video/SingleVideo.css:28]: will try to implement these suggestion in next pr,

**Pull request merged**